PocketKit/Sources/PocketKit/Home/SlateDetailViewController.swift

```diff
@@ -253,7 +222,7 @@ private extension SlateDetailViewController {
 
             if let action = self.model.reportAction(for: viewModelCell, at: indexPath),
                let uiAction = UIAction(action) {
-                cell.overflowButton.addAction(uiAction, for: .primaryActionTriggered)
+                cell.overflowButton.menu = UIMenu(children: [uiAction])
             }
```

Comment:
I know the code that came before was setting the action here in the view controller, but I really would prefer to move it into the cell if possible. (similar idea to what I was getting at w/ HomeViewController et al)